### PR TITLE
Add PodSecurityPolicy creation and binding.

### LINKED ISSUE #292: Automatically install a podsecurity policy during direct-csi install

### PR DESCRIPTION
**Pull request by balamurugana**
Fixes #292

Signed-off-by: Bala.FA <bala.gluster@gmail.com>

### CODE REVIEW

**Review by wlan0 — APPROVED** [review]

**Comment by wlan0** [comment]
LGTM

File: pkg/utils/installer/install.go [hunk 1]
```diff
@@ -431,6 +431,10 @@ func CreateDaemonSet(ctx context.Context, identity string, directCSIContainerIma
 				}(),
 				SecurityContext: &corev1.SecurityContext{
 					Privileged: &privileged,
+					SeccompProfile: &corev1.SeccompProfile{
```
Comment (Praveenrajmani) [inline pkg/utils/installer/install.go:434]: Should we enable Seccomprofile by default?

File: pkg/utils/installer/install.go [hunk 2]
```diff
@@ -509,6 +513,7 @@ func CreateDaemonSet(ctx context.Context, identity string, directCSIContainerIma
 					Namespace: sanitizeName(name),
 					Annotations: map[string]string{
 						CreatedByLabel: DirectCSIPluginName,
+						"container.apparmor.security.beta.kubernetes.io/direct-csi": apparmorProfileName,
```
Comment (Praveenrajmani) [inline pkg/utils/installer/install.go:516]: Should we enable apparmour by default?

**Review by Praveenrajmani — COMMENTED** [review]
Can we maintain flags for seccomp, apparmour and selinux? So that, only when we any of these flags,  we can go and enable it in the containers.

File: pkg/utils/installer/install.go (hunk 1 above)
Comment (balamurugana) [reply pkg/utils/installer/install.go:434]: No. Made it optional now.

File: pkg/utils/installer/install.go (hunk 2 above)
Comment (balamurugana) [reply pkg/utils/installer/install.go:516]: No. Made it optional now.

**Review by wlan0 — APPROVED** [review]

**Pull request merged**